recipe/meta.yaml

```diff
@@ -24,11 +24,15 @@ requirements:
     - python
     - setuptools
     - toolchain
+    - libcxx  # [osx]
```

Comment:
Would add `clangdev` for macOS above as that is going to be needed to build with `libcxx` and `openmp`. Though it is getting pulled in automatically ATM. That probably won't be the case going forward.